android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModuleActivity.java

```diff
@@ -0,0 +1,58 @@
+package com.reactnative.ivpusic.imagepicker;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.provider.MediaStore;
+import android.support.annotation.Nullable;
+
+import com.reactnative.ivpusic.imagepicker.activity.BasePickerActivity;
+
+/**
+ * Created by song on 2018/1/6.
+ */
+
+public class PickerModuleActivity extends BasePickerActivity {
+    static final String REQUEST_CODE_KEY = "requestCode";
+    static final String IS_PICK_KEY = "pick";
+    static final String URI_KEY = "uri";
```

Comment:
Why not just use
+import static com.reactnative.ivpusic.imagepicker.PickerModuleActivity.IS_PICK_KEY;
+import static com.reactnative.ivpusic.imagepicker.PickerModuleActivity.REQUEST_CODE_KEY;
+import static com.reactnative.ivpusic.imagepicker.PickerModuleActivity.URI_KEY;